frontend/app/scripts/controllers/login.js

```diff
@@ -2,29 +2,50 @@
  * Created by Natsik on 15.07.2015.
  */
 angular.module('frontendApp')
-  .controller('LoginCtrl', function ($scope,
-                                     $localStorage) {
+  .controller('LoginCtrl',['$scope', '$cookies', '$base64', '$localStorage', '$rootScope', '$http', '$location', 'AuthenticationService',
+    function ($scope, $cookies, $base64, $localStorage, $rootScope, $http, $location, AuthenticationService) {
     $scope.$storage = $localStorage;
 
-    user.email = $localStorage.email || "example@gmail.com";
-    this.user = user;
+    $scope.credentials = {};
 
-    this.login = function () {
-      alert("This functionality in progress");
-
-      var rememberMeCheckbox = document.getElementById("rememberMe");
-      if (rememberMeCheckbox && rememberMeCheckbox.type == "checkbox" && rememberMeCheckbox.checked) {
-        $localStorage.email = this.user.email;
-      }
-      //TODO clear form after success
-    }
-
-    this.clearLocalStorage = function () {
-      delete $localStorage.email;
+    $scope.login = function () {
+      $scope.dataLoading = true;
+      AuthenticationService.Login($scope.credentials, function (response) {
+        if (response!=null) {
+          AuthenticationService.SetCredentials(response.name, response.details.sessionId, true);
+          if($rootScope.locationPath!="/login"){
```

Comment:
You should use strick check !== for compare strings.
Add also white space please.